Clean/Circuit/Expression.lean

```diff
@@ -15,6 +15,30 @@ inductive Expression (F : Type) where
   | add : Expression F -> Expression F -> Expression F
   | mul : Expression F -> Expression F -> Expression F
 
+
+partial def exprToJson : Expression F → Lean.Json
+  | .var v   => Lean.Json.mkObj [
+                  ("type", Lean.Json.str "var"),
+                  ("index",     Lean.Json.num v.index)   -- Nat’s instance
+                ]
```

Comment:
can you consistently use an indentation of 2 spaces?
also, there should be just one space in between symbols.

here for example, I would expect:
```suggestion
  | .var v => Lean.Json.mkObj [
    ("type", Lean.Json.str "var"),
    ("index", Lean.Json.num v.index) -- Nat’s instance
  ]
```

Reply:
maybe we should have a standard for auto formatting?

Reply:
that would be nice, I don't think an established standard exists for Lean yet. I remember this was in the priorities for Lean language development this year